src/ai/ai_config.cpp

```diff
@@ -118,3 +118,13 @@ void AIConfig::SetSetting(const char *name, int value)
 
 	ScriptConfig::SetSetting(name, value);
 }
+
+void AIConfig::AddRandomDeviation()
+{
+	int start_date = this->GetSetting("start_date");
+
+	ScriptConfig::AddRandomDeviation();
+
+	/* start_date = 0 is a special case, where random deviation does not occur. */
```

Comment:
This comment doesn't explain why you are resetting start_date regardless of it being 0 or not. Can you please elaborate on this in your comment.